### PR TITLE
Update direct-routing-plan-media-bypass.md

### PR DESCRIPTION
**Pull request by NMuravlyannikov**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan-media-bypass.md [hunk 1]
```diff
@@ -229,7 +229,7 @@ You need to open ports 3478 and 3479 for transitioning. When Microsoft introduce
 
 ### Requirements for using media processors
 
-Media Processors are always in the media path for voice applications. The requirements are the same as for non-bypass configuration.
+Media Processors are always in the media path for voice applications and for Web cleints (for exampe, Teams cleint in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
Media Processors are always in the media path for voice applications and for Web clients (for example, Teams clients in Edge or Google Chrome). The requirements are the same as for non-bypass configuration.
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 2]
```diff
@@ -277,9 +277,9 @@ For information about how to configure two trunks on the same SBC, see the docum
 
 ## Client endpoints supported with media bypass
 
-Media bypass is supported with all Teams endpoints, except Teams Web clients until further notice. 
+Media bypass is supported with all Teams endpoints.
 
-If your users prefer Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox, the media bypass for such users must be turned off. We will introduce calling using a media bypass enabled trunk in the future.   
+Note for webcleints (Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox) we will covert the call to non-bypass even of it started as a bypass call. This happens automatically and does not require any actions on the administrator.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md]:
```suggestion
> [!NOTE]
> The call will be converted to non-bypass even if it started as a bypass call for web clients using the Teams Web app in Microsoft Edge, Google Chrome or Mozilla Firefox. This happens automatically and does not require any actions from the administrator. 
```

**Comment by LolaJacobsen** [comment]
@CarolynRowe - for review

**Pull request merged**